app/helpers/application_helper.rb

```diff
@@ -34,4 +34,16 @@ def decorated_session
       @_decorated_session ||= SessionDecorator.new
     end
   end
+
+  def sp_session
+    session[:sp]
+  end
+
+  def loa1_signup_context?
+    sp_session && current_user && !current_user.recovery_code.present?
```

Comment:
Yes, per this comment https://github.com/18F/identity-private/issues/1333#issuecomment-277835063.

Currently we are only showing these links on a subset of the pages they'll appear on, there are other open issues to address getting the correct cancel links in their proper spots.